tests/sass/space-around-operator.sass

```diff
@@ -0,0 +1,405 @@
+//====================
+// Should fail
+//====================
+
+// FIXME: Gonzales issue
+// https://github.com/tonyganch/gonzales-pe/issues/85
+// .foo
+//   background: url('image'  +  '/foo.png')
+//
+
```

Comment:
Should we not add a indented syntax mixin here seen as they use similar characters? 

```sass
=large-text
  font:
    family: Arial
    size: 20px
    weight: bold
  color: #ff0000

h1
  +large-text
```